src/GAP.jl

```diff
@@ -253,8 +255,16 @@ function __init__()
 
     real_JuliaInterface_path[] = Setup.locate_JuliaInterface_so(sysinfo)
 
-    gaproots = sysinfo["GAPROOTS"]
-    cmdline_options = ["", "-l", gaproots]
+    gaproot_gapjl = abspath(@__DIR__, "..")
+    gap_lib_dir = abspath(GAP_lib_jll.find_artifact_dir(), "share", "gap")
+    mkpath(Packages.DEFAULT_PKGDIR[])
+    roots = [
+            gaproot_gapjl,      # for JuliaInterface and JuliaExperimental
+            Packages.DEFAULT_PKGDIR[],
```

Comment:
`Packages.DEFAULT_PKGDIR[]` is a path with suffix `/pkg`. The corresponding root directory is the path without this suffix.

Reply:
```suggestion
            abspath(Packages.DEFAULT_PKGDIR[], ".."),
```

Reply:
Alternatively, we could pass the `DEFAULT_PKGDIR[]` via `--packagedirs` instead of `-l`. The same would be possible for `gaproot_gapjl` as well, since this is only included in the list for providing the `pkg/` subdir.

Reply:
Yes, eventually the packages should be handled via package directories, not root directories.
(I was not sure whether the current GAP.jl must still support the "old" setup where we cannot assume package directories independent of root directories.)

Reply:
Since GAP_lib_jll 400.1400.003 the PR to gap is included, so we can assume this.